tests/integration/dhcp_test.py

```diff
@@ -219,3 +219,79 @@ def _clean_up():
         os.unlink(DNSMASQ_CONF_PATH)
     except FileNotFoundError:
         pass
+
+
+def test_slave_ipaddr_learned_via_dhcp_added_as_static_to_linux_bridge(dhcp_env
```

Comment:
If you use `ip` instead of `ipaddr`, will it fit one line?

Reply:
yes, fixed